Homework_5/hw_task_2.py

```diff
@@ -0,0 +1,16 @@
+"""Напишите однострочный генератор словаря, который принимает на вход три списка
+одинаковой длины: имена str, ставка int, премия str с указанием процентов вида “10.25%”.
+В результате получаем словарь с именем в качестве ключа и суммой премии в качестве значения.
+Сумма рассчитывается как ставка умноженная на процент премии"""
+
+
+def get_dict(names, rates, percents):
+    return {i[0]: i[1] * float(i[2].strip('%')) / 100 for i in zip(names, rates, percents)}
```

Comment:
Можно не использовать индексирование, а сразу помещать в нужные перемены?
```suggestion
    return {name: rate * float(percent [:-1]) / 100 for name, rate, percent in zip(names, rates, percents)}
```